appservice/src/createAppService/AppServicePlanRedundancyStep.ts

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IAzureQuickPickItem } from '@microsoft/vscode-azext-utils';
+import { localize } from '../localize';
+import { IAppServiceWizardContext } from './IAppServiceWizardContext';
+import { AzExtLocation } from '@microsoft/vscode-azext-azureutils';
+
+interface AppServiceWizardContext extends IAppServiceWizardContext {
+    _location: AzExtLocation;
+    zoneRedundant: boolean;
+}
+
+export class AppServicePlanRedundancyStep extends AzureWizardPromptStep<IAppServiceWizardContext> {
+    public async prompt(context: AppServiceWizardContext): Promise<void> {
+        const placeHolder: string = localize('selectZoneRedundancy', 'Select zone redundancy availability');
+        const picks: IAzureQuickPickItem<boolean>[] = [
+            { label: localize('enabled', 'Enabled'), data: true },
+            { label: localize('disabled', 'Disabled'), data: false }
+        ];
+
+        context.zoneRedundant = (await context.ui.showQuickPick(picks, { placeHolder })).data;
+    }
+
+    // TODO(ccastrotrejo): This will be changed to use orgdomain with WI 12845265 once georegions API is updated with ANT78.
+    private isZoneRedundancyEnabled(location: string): boolean {
+        const zoneRedundancySupportedLocations = [
+            'westus2',
+            'westus3',
+            'centralus',
+            'eastus',
+            'eastus2',
+            'canadacentral',
+            'brazilsouth',
+            'northeurope',
+            'westeurope',
+            'germanywestcentral',
+            'francecentral',
+            'uksouth',
+            'japaneast',
+            'southeastasia',
+            'australiaeast',
+            'eastus2euap',
+        ];
+
+        return zoneRedundancySupportedLocations.includes(location);
+    }
+
+    public shouldPrompt(context: AppServiceWizardContext): boolean {
+        const { customLocation, _location, plan } = context;
+        if (plan === undefined && customLocation === undefined && _location && _location.name) {
```

Comment:
According to the [requirements in the docs](https://docs.microsoft.com/en-us/azure/app-service/how-to-zone-redundancy#requirements), zone redundancy:
> Requires either Premium v2 or Premium v3 App Service plans

This step should not be shown unless the user selects a compatible App Service plan. If the ASP is not compatible, then zone redundancy should be disabled.

Reply:
Yes, you are right. Missing the decision over there. Thanks!

Reply:
Testing it again now